lib/gpx.js

```diff
@@ -191,6 +191,12 @@ export function* gpxGen(doc) {
 export function gpx(doc) {
   return {
     type: "FeatureCollection",
-    features: Array.from(gpxGen(doc))
+    features: Array.from(gpxGen(doc)),
+    properties: gpxProperties(doc),
   };
 }
+
+export function gpxProperties(doc) {
+  const gpx = doc.getElementsByTagName("gpx");
+  return gpx && gpx.length ? getProperties(gpx[0]) : {};
```

Comment:
As far as I can tell, this would incorrectly include all `<link>` elements in the gpxProperties, even if those elements were inside of individual features, not just the top-level gpx element.

Reply:
Additionally, the required & optional properties of the top-level gpx element are different than those on features - https://www.topografix.com/gpx_manual.asp#gpx_req - so this should probably not reuse the existing getProperties function.